routes/api/comments.js

```diff
@@ -0,0 +1,65 @@
+const createError = require("http-errors");
+const express = require("express");
+
+const { asyncHandler } = require("../utils");
+const { CommentLike } = require("../../db/models");
+const { restoreUser } = require("../../auth");
+
+const router = express.Router();
+
+/* Like a comment */
+router.post(
+  "/comments/:id/likes",
+  restoreUser,
+  asyncHandler(async (req, res, next) => {
+    if (!res.locals.authenticated) {
+      /* Return error 401, if
+        user is not authenticated, or
+        user is not liker */
+      res.status(401);
```

Comment:
Utilize error handlers for error status codes. See previous examples you've implemented in this branch.

Reply:
Utilized error handler